tests/foobar/abc.js

```diff
@@ -0,0 +1,5 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
where do these `foobar` files come from? is this TS specific?

Reply:
also seem related to https://github.com/google/flatbuffers/pull/7161 and specifically this included schema - https://github.com/google/flatbuffers/blob/master/tests/test_dir/typescript_transitive_include.fbs

Reply:
Yeah, these don't have to be saved in the repo. The `tests/TypeScripttest.sh` should output them to a temp file, assert they exist, and then delete them. If we do that, we don't have to worry about checking them in.